packages/website/src/pages/index.tsx

```diff
@@ -26,9 +26,9 @@ const features: FeatureItem[] = [
             />
           </div>
           <div className="col col--8">
-            <h4 className="text--justify">
+            <h3 className="text--justify">
               <b>ESLint</b> is an awesome linter for JavaScript code.
-            </h4>
+            </h3>
```

Comment:
Before, the page went `h1 > h2 > h4`. You can't jump up more than 1 heading level at a time.

https://dequeuniversity.com/rules/axe/4.0/heading-order

In theory I could have applied a CSS style to keep these at the same size as before, but I kinda like them slightly bigger now...